view/frontend/web/js/store/index.js

```diff
@@ -51,7 +51,8 @@ const store = new Vuex.Store({
       shippingCarrierCode: '',
       shippingMethodCode: ''
     },
-    totals: null
+    totals: {},
```

Comment:
why you changed to empty object?

Reply:
lol idk

Reply:
And want to see it it working or no. For now we are not setting to totals anything. Is it ok?

Reply:
yes, for now we don't use this state